shimmer/learn/governance/shimmer-firefly-governance-guide.md

```diff
@@ -55,6 +55,9 @@ To vote on a proposal, you must tell the Wallet how many SMR tokens you want to
 
 The amount of SMR tokens you choose for governance are now transferred to a separate output and can only be used for governance now. If you want to send them out of your Wallet, or use them for anything else you will have to remove them from the governance output by reducing your voting power.
 
+Important Information:
+- This voting output belongs to your own private key! By voting, you are never losing control over your Tokens. The voting output is just a specified output that is only used for your governance activities but the tokens never leave your control in this process.
+
```

Comment:
This is the perfect place for an [admonitions](https://docusaurus.io/docs/markdown-features/admonitions)
Feel free to suggest another title, it's just something I came up with real quick.

```suggestion
::: info Token Ownership
The voting output belongs to your own private key! By voting, you are never losing control over your Tokens. The voting output is just a specified output that is only used for your governance activities but the tokens never leave your control in this process.
```